public/scripts/timeline.js

```diff
@@ -1395,6 +1398,9 @@ $(function(){
     });
 
     $("#updateMementos").click(function(event){
+
+        let chosenMementos = JSON.parse(JSON.stringify(jsonObjRes));

```

Comment:
Why are we first stringifying the object and then parsing it back?

Reply:
It's creating a deep copy of the list of mementos so the user can delete mementos but also maintain the original list.

Reply:
Makes sense, thanks!